src/main/java/com/example/icebutler_server/fridge/dto/response/FridgeUserRes.java

```diff
@@ -1,36 +1,31 @@
 package com.example.icebutler_server.fridge.dto.response;
 
+import com.example.icebutler_server.fridge.entity.FridgeUser;
 import com.example.icebutler_server.global.entity.FridgeRole;
 import com.example.icebutler_server.global.util.AwsS3ImageUrlUtil;
-import com.example.icebutler_server.user.entity.User;
 import io.swagger.v3.oas.annotations.media.Schema;
-import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
-import lombok.NoArgsConstructor;
 
 @Data
-@AllArgsConstructor
-@NoArgsConstructor
 @Builder
-@Schema(name = "FridgeUserRes", description = "냉장고 유저 정보")
+@Schema(name = "냉장고 유저 정보", description = "FridgeUserRes")
 public class FridgeUserRes {
-    @Schema(name = "userId", description = "유저 ID")
+    @Schema(description = "유저 ID", example = "1")
     private Long userId;
-    @Schema(name = "nickname", description = "유저 닉네임")
+    @Schema(description = "유저 닉네임", example = "나야나")
     private String nickname;
-    @Schema(name = "role", description = "냉장고 내 유저 역할")
+    @Schema(description = "냉장고 내 유저 역할", example = "OWNER/MEMBER")
```

Comment:
```suggestion
    @Schema(description = "냉장고 내 유저 역할", allowableValues = {"OWNER", "MEMBER"})
```

스웨거에서 `allowableValues` 사용하면 좀 더 명확하게 명세할 수 있을 것 같아요!
[관련설명링크](https://blog.jiniworld.me/91)